test/e2e/e2e_suite_test.go

```diff
@@ -161,6 +171,62 @@ var _ = SynchronizedBeforeSuite(func() []byte {
 		Expect(err).NotTo(HaveOccurred())
 	}
 
+	// Metrics test start
+	By("creating a ClusterRoleBinding for the service account to allow access to metrics")
+	cmd := exec.Command("kubectl", "create", "clusterrolebinding", metricsRoleBindingName,
+		"--clusterrole=baremetal-operator-metrics-reader",
+		fmt.Sprintf("--serviceaccount=%s:%s", namespace, serviceAccountName),
+	)
+	_, err := utils.Run(cmd)
```

Comment:
This function uses os.Chdir() to change directory to repo root (https://github.com/kubernetes-sigs/kubebuilder/blob/master/testdata/project-v4/test/utils/utils.go#L48).

Originally the repo is set to tools/bin (where ginkgo binary locates), so the kustomizations in the config are set as "../../config/...".

Reply:
that was the issue , thanks, I have pushed a fix.